README.md

```diff
@@ -125,8 +125,13 @@ read:
 write:
   pragmas: <list of pragma settings for read/write>
 ```
-Please refer to [documentation of pragmas](https://www.sqlite.org/pragma.html).
+
+By default, SQLite settings are significantly optimized for performance.
+This might have consequences of bag data being corrupted after an application or system-level crash.
+If increased crash-caused corruption resistance is necessary, use `--resilient-storage-writing` flag.
```

Comment:
Should note that `any values specified in the `--storage-config-file` take precedence over default or preset values.